bom/application/pom.xml

```diff
@@ -1292,6 +1292,19 @@
                 <artifactId>quarkus-jdbc-derby-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jms-spi</artifactId>
+                <version>${project.version}</version>
```

Comment:
@turing85 I think we don't have `io.quarkus:quarkus-jms-spi`.

Reply:
true. Removed (also for the companion PR).